### PR TITLE
Add and link to tool homepages

### LINKED ISSUE #245: Link to bundler project sites somewhere
@argyleink any thoughts on where to put them?

### PR DESCRIPTION
**Pull request by awayken**
Fixes #245.

### CODE REVIEW

File: shared/utils/tool-homepages.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const browserify = 'http://browserify.org/';
+export const rollup = 'https://rollupjs.org/';
+export const webpack = 'https://webpack.js.org/';
+export const parcel = 'https://parceljs.org/';
```
Comment (surma) [inline shared/utils/tool-homepages.ts]: We should probably integrate this into `config.js`. 

I’d say let’s open an issue and do that in another PR as that one will have wide-reaching changes.

**Review by surma — APPROVED** [review]
LGTM, gonna wait on @argyleink and/or @una to review tho

**Review by una — APPROVED** [review]

File: shared/utils/tool-homepages.ts (hunk 1 above)
Comment (devongovett) [reply shared/utils/tool-homepages.ts]: Maybe link to v2.parceljs.org rather than the v1 site, since that's what's tested on tooling.report?

Comment (surma) [reply shared/utils/tool-homepages.ts]: Good shout!

Comment (surma) [reply shared/utils/tool-homepages.ts]:
```suggestion
export const parcel = 'https://v2.parceljs.org/';
```

Comment (surma) [reply shared/utils/tool-homepages.ts]: Filed #383
